src/api/wallet/composeProvider.ts

```diff
@@ -62,6 +62,47 @@ function providerAsMiddleware(provider: Provider): JsonRpcMiddleware {
   }
 }
 
+// wait 7 sec, for now
+const DEFAULT_TX_APPROVAL_TIMEOUT = 7000
```

Comment:
nice for the test, although for real, at least we need sth like 1min, right?

Reply:
of course :+1: